<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPreferencesPageLocations</class>
 <widget class="QWidget" name="WdgPreferencesPageLocations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <widget class="QListWidget" name="lwLocationCategories"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="swLocationSelector">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pSingleSelection">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="lLocationSingleFolder">
         <property name="text">
          <string>Folder:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="leLocationSingleFolder">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbBrowseLocationSingleFolder">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../Resources.qrc">
             <normaloff>:/Icons/folder-open.ico</normaloff>:/Icons/folder-open.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>417</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pMultipleSelection">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="lLocationMultipleFolders">
         <property name="text">
          <string>Folders:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <widget class="QListWidget" name="lwLocationMultipleFolders"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pbLocationMultipleFoldersAdd">
             <property name="text">
              <string>Add</string>
             </property>
             <property name="icon">
              <iconset resource="../../Resources.qrc">
               <normaloff>:/Icons/plus.ico</normaloff>:/Icons/plus.ico</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbLocationMultipleFoldersRemove">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset resource="../../Resources.qrc">
               <normaloff>:/Icons/Remove.ico</normaloff>:/Icons/Remove.ico</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
